Documentation/docs.md

```diff
@@ -0,0 +1,41 @@
+# Documentation
+
+etcd is a distributed key-value store designed for serving most critical data. It enables reliable distributed coordination, like distributed locking, leader election and barrier. Once an etcd cluster is set up, it is supposed to be there for you "forever".
```

Comment:
etcd is a distributed key-value store designed to reliably and quickly preserve and provide access to critical data. It enables reliable distributed coordination of this data, through distributed locking, leader elections, and write barriers. An etcd cluster is intended for high availability and permanent data storage and retrieval.